docs/migrate-from-crate-client.md

```diff
@@ -1,54 +1,50 @@
 (migrate-from-crate-python)=
 # Migrate from `crate.client`
 
+## Introduction
 In June 2024, code from the package [crate\[sqlalchemy\]] has been transferred
 to the package [sqlalchemy-cratedb]. For 80% of use cases, this will be
-a drop-in replacement with no noticeable changes.
-
-However, if you use CrateDB's special data types like `OBJECT`, `ARRAY`,
-`GEO_POINT`, or `GEO_SHAPE`, and imported the relevant symbols from
-`crate.client.sqlalchemy`, you will need to import the same symbols from
-`sqlalchemy_cratedb` from now on.
+a drop-in replacement with no noticeable changes. In November 2024,
+`crate-1.0.0` has been released, which dropped including the SQLAlchemy
```

Comment:
> which dropped including

Not super clear, maybe could be rephrased?

Reply:
Sure, thanks. @kneth: Coming up with improvement proposals right away will tremendously help.

Reply:
Maybe something like

> With the release of `crate-1.0.0` in November 2024, the support of the SQLAlchemy dialect was dropped. See the "Upgrade procedure" section for further information.

Reply:
Thank you very much. I've used your suggestion 1:1, and compressed it by a few more words.

> With the release of `crate-1.0.0` in November 2024, the SQLAlchemy dialect was
dropped from this package. See the "Upgrade procedure" section for further
information.